cartography/intel/aws/iam.py

```diff
@@ -271,6 +271,11 @@ def get_role_list_data(boto3_session: boto3.session.Session) -> Dict:
         try:
             role_data = client.get_role(RoleName=role.get("RoleName")).get("Role")
             role['RoleLastUsed'] = role_data.get("RoleLastUsed")
+
+            if role_data.get('Path', '').startswith('/aws-service-role/'):
+                #Skip this roles from IAM-JIT, because we can't edit the trust policy for this roles
+                role['isServiceRole'] = True
```

Comment:
instead of calling `isServiceRole`, let's call it `isPredefinedRole`

Reply:
edit trust policy issue with only service roles not for all predefined roles, so i think good way is calling isServiceRole